### PR TITLE
Sai/table name bug fix

### PR DESCRIPTION
**Pull request by svv232**

### CODE REVIEW

File: src/skmysql.ml [hunk 1]
```diff
@@ -1038,7 +1038,8 @@ module Table = struct
   let pp_column_name fmt (Column.Pack c) = Column.pp_spec_name fmt c
 
   let pp_column fmt column =
-    Fmt.pf fmt "@[%a %a%a@]" pp_column_name column pp_column_type column
+    (* Adding backticks to column name so columns can be named mysql key words *)
+    Fmt.pf fmt "@[`%a` %a%a@]" pp_column_name column pp_column_type column
```
Comment (zbaylin) [inline src/skmysql.ml:1042]: it scares me that this wasn't here before

**Review by zbaylin — APPROVED** [review]
Looks good to me :)

**Review by UnrealAkama — APPROVED** [review]
This looks like a nice change. One thing we should confirm though is pin this branch for the main repo and make sure basic stuff is working correctly before we merge it in.

File: src/skmysql.ml (hunk 1 above)
Comment (bleepbloopsify) [reply src/skmysql.ml:1042]: >just dont name a column "from" haha

but same

**Review by bleepbloopsify — APPROVED** [review]

**Pull request merged**